mozmeao-fr/basket-stage/web-deploy.yaml

```diff
@@ -38,6 +38,42 @@ spec:
         - /bin/bash
         - -c
         env:
+        - name: ACOUSTIC_CLIENT_ID
+          valueFrom:
+            secretKeyRef:
+              key: acoustic-client-id
+              name: basket-prod-secrets
+        - name: ACOUSTIC_CLIENT_SECRET
+          valueFrom:
+            secretKeyRef:
+              key: acoustic-client-secret
+              name: basket-prod-secrets
+        - name: ACOUSTIC_REFRESH_TOKEN
+          valueFrom:
+            secretKeyRef:
+              key: acoustic-refresh-token
+              name: basket-prod-secrets
```

Comment:
these should all be `basket-stage-secrets`

Reply:
Okay. I wasn't sure the preference here. I can see liking the visibility that these settings are the same as prod. But it is crossing the streams a bit.

Reply:
I believe it simply won't work. The secrets are added to their respective namespaces, so configs in the stage namespace won't have access to secrets in the prod one.